mt940/tags.py

```diff
@@ -190,17 +190,32 @@ class NonSwift(Tag):
 
     Pattern: 2!n35x
     '''
+
+    class scope(models.Transaction, models.Transactions):
+        pass
     id = 'NS'
+
     pattern = r'''
     (?P<non_swift>
-        (\d{2}.{0,35})
-        (\n\d{2}.{0,35})*
+        (\d{2}.{0,50})
+        (\n\d{2}.{0,50})*
     )
     $'''
     sub_pattern = r'''
-    (?P<ns_id>\d{2})(?P<ns_data>.{0,35})
+    (?P<ns_id>\d{2})(?P<ns_data>.{0,48})
     '''
 
+    def strip_non_swift(self, data):
+        return '\n'.join((l for l in (line[2:] for line in data.split('\n')) if l))
```

Comment:
This line fails the pep8 check because it's a bit too long. Perhaps we can convert it to a regular loop? Might be a bit more readable as well.